xml/System.Security.Cryptography.Cose/CoseHeaderMap.xml

```diff
@@ -351,6 +353,7 @@
         <param name="arrayIndex">The zero-based index in <paramref name="array" /> at which copying begins.</param>
         <summary>Copies the elements of the <see cref="T:System.Collections.Generic.ICollection`1" /> to an <see cref="T:System.Array" />, starting at a particular <see cref="T:System.Array" /> index.</summary>
         <remarks>To be added.</remarks>
+        <inheritdoc />
```

Comment:
I suspect all the inheritdoc in this file failed because the type inherits from two interfaces. And even if one interface defines a method, but not the other interface, mdoc isn't smart enough to decide where to look.

I'll try to prove my hypothesis by adding crefs:

```suggestion
        <inheritdoc cref="M:System.Collections.Generic.ICollection`1.CopyTo(`0[],System.Int32)" />
```